src/components/Dropdown/VersionSelect.tsx

```diff
@@ -439,9 +439,6 @@ function generateArchivedWebsiteUrlByLangAndType(lang?: string, type?: string) {
     case "tidb-in-kubernetes":
       url = `${url}/tidb-in-kubernetes/v1.0`;
       break;
-    case "tidb-data-migration":
-      url = `${url}/tidb-data-migration/v1.0`;
-      break;
```

Comment:
Do we need to keep the above as it is for the archived doc site?  https://docs-archive.pingcap.com/zh/tidb-data-migration/v1.0

Reply:
It has no impact on the archived site. This is used for navigating from the docs site to the archived site.